src/query/__tests__/RelayQueryRoot-test.js

```diff
@@ -342,6 +342,77 @@ describe('RelayQueryRoot', () => {
     expect(meIdentifyingArg).toBeUndefined();
   });
 
+  it('returns the identifying number argument ', () => {
+    var nQuery = getNode(Relay.QL`
+      query {
+        task(number:5){
+          dumb,
+        }
+      }
+    `);
+    nQuery.getConcreteQueryNode().metadata= {
+      identifyingArgName: 'number',
+      identifyingArgType: 'scalar' };
```

Comment:
@josephsavona Get it, thanks for your help ,i think i should finish this PR tonight, this PR stimulate me to planning some times to learn the right work process for Javascript. I think if i learned (jest and babel-plugins ) before, this pull request should be much simpler.  ~_~ at night, i should make ESlint work right in webstrom first, then i can automatic do the right style,when inputting characater.